src/index.js

```diff
@@ -0,0 +1,35 @@
+import './style.css';
+
+const tasks = [
+  {
+    id: 1,
+    task: 'Wash my car',
+    completed: false,
+  },
+  {
+    id: 2,
+    task: 'Shopping for my child',
+    completed: true,
+  },
+  {
+    id: 3,
+    task: 'Microverse Project',
+    completed: false,
+  },
+];
+
+const toDoLists = document.getElementById('todoList');
+
+function AllTasks() {
+  let chore = '';
+  tasks.forEach((alltask) => {
+    chore += `
+      <div class="list">
+        <input type="checkbox">
+        <p>${alltask.task}</p>               
+        <a><i class="fa fa-ellipsis-v fa-2x" aria-hidden="true"></i></a>
+      </div>`;
+  });
+  toDoLists.innerHTML = chore;
+}
+AllTasks();
```

Comment:
- You have done a good job here 👍. But please note that you are required, according to the project requirements, to dynamically render your page when the DOM loads. I suggest you try using an event listener that listens to the `DOMContentLoaded` . You may find this [article](https://www.w3schools.com/js/js_htmldom_eventlistener.asp) in solving this issue. 😄